http/analysis-service/crontab.py

```diff
@@ -7,8 +8,7 @@
 #### YOUR CHANGES WILL BE OVERWRITTEN ####"""
 CRON_END      = "#--END TELEMETRY SCHEDULED JOBS--"
 CRONTAB_CMD = "/usr/bin/crontab"
-# Yuck - it's very ugly to have the path specified here :(
-CRON_RUNNER_LOCATION = "/home/ubuntu/telemetry_analysis/jobs/run.sh"
+CRON_RUNNER_LOCATION = os.path.dirname(os.path.realpath(__file__)) + "/jobs/run.sh"
```

Comment:
Just to confirm, this moves the "job runner" path to `/home/ubuntu/telemetry-server/http/analysis-service/jobs/run.sh` instead of in `/home/ubuntu/telemetry_analysis/jobs/run.sh`, right?